content/functions/math.md

```diff
@@ -20,15 +20,16 @@ draft: false
 aliases: []
 ---
 
-| Function     | Description                                                                 | Example                        |
-|--------------|-----------------------------------------------------------------------------|--------------------------------|
-| `add`        | Adds two integers.                                                          | `{{add 1 2}}` &rarr; 3         |
-| `div`        | Divides two integers.                                                       | `{{div 6 3}}` &rarr; 2         |
-| `mod`        | Modulus of two integers.                                                    | `{{mod 15 3}}` &rarr; 0        |
-| `modBool`    | Boolean of modulus of two integers. Evaluates to `true` if result equals 0. | `{{modBool 15 3}}` &rarr; true |
-| `mul`        | Multiplies two integers.                                                    | `{{mul 2 3}}` &rarr; 6         |
-| `sub`        | Subtracts two integers.                                                     | `{{sub 3 2}}` &rarr; 1         |
-|--------------|-----------------------------------------------------------------------------|--------------------------------|
```

Comment:
@moorereason did you add the divider because floor, ceil and round are a logical unit or because those template funcs require the package name in order to be called?

If it's the latter case I'll move `math.Log` there as well.

Reply:
I put it in there to see how it looked but actually meant to delete it before committing!  You can do what you want with it.

Reply:
As divider not intended to provide any further meaning I'll remove it.